ansible/roles/ecryptfs/tasks/main.yml

```diff
@@ -34,7 +34,7 @@
 
 - name: Register mount configuration
   sudo: yes
-  shell: "tail /etc/mtab -n1"
+  shell: "cat /etc/mtab | grep '/opt/data' | sed -e 's/ rw/ user,noauto,rw/'"
```

Comment:
oh i see. that is tricky. seems like `sed` is the easiest solution. it also seems odd though that it isn't there already when it adds `user,noauto` when you mount: https://github.com/dimagi/commcarehq-ansible/blob/ecryptfs/ansible/roles/ecryptfs/tasks/main.yml#L22